Description: Adopt cmake file for the Debian and derivatives
  We do not want to put libexec into the /usr. But into /usr/lib/
Author: Anton Gladky <gladk@debian.org>
Last-Update: 2018-08-21

Index: kim-api/cmake/Modules/FindKIM-API-V2.cmake.in
===================================================================
--- kim-api.orig/cmake/Modules/FindKIM-API-V2.cmake.in
+++ kim-api/cmake/Modules/FindKIM-API-V2.cmake.in
@@ -58,7 +58,7 @@ else()
   set(KIM_API_VERSION @PROJECT_VERSION@)
 
   set(KIM_API_CMAKE_DIR @CMAKE_INSTALL_FULL_LIBDIR@/@PROJECT_NAME@/@KIM_API_CMAKE_DIR_IDENTIFIER@)
-  set(KIM_API_COLLECTIONS_INFO_EXECUTABLE @CMAKE_INSTALL_FULL_LIBEXECDIR@/@PROJECT_NAME@/@PROJECT_NAME@-collections-info)
+  set(KIM_API_COLLECTIONS_INFO_EXECUTABLE /usr/lib/libexec/@PROJECT_NAME@/@PROJECT_NAME@-collections-info)
 
 
   set(KIM_API_MODEL_DRIVER_IDENTIFIER @KIM_API_MODEL_DRIVER_IDENTIFIER@ CACHE INTERNAL "Model driver id")
Index: kim-api/examples/model-drivers/LennardJones612__MD_414112407348_003/CMakeLists.txt
===================================================================
--- kim-api.orig/examples/model-drivers/LennardJones612__MD_414112407348_003/CMakeLists.txt
+++ kim-api/examples/model-drivers/LennardJones612__MD_414112407348_003/CMakeLists.txt
@@ -39,7 +39,7 @@
 
 cmake_minimum_required(VERSION 3.4)
 
-list(APPEND CMAKE_MODULE_PATH $ENV{KIM_API_V2_CMAKE_MODULES_DIR})
+list(APPEND CMAKE_MODULE_PATH $ENV{KIM_API_V2_CMAKE_MODULES_DIR} "/usr/share/kim-api-v2/cmake")
 find_package(KIM-API-V2 2.0 REQUIRED)
 if(NOT TARGET kim-api)
   project("${KIM_API_PROJECT_NAME}" VERSION "${KIM_API_VERSION}"
Index: kim-api/examples/model-drivers/ex_model_driver_P_LJ/CMakeLists.txt
===================================================================
--- kim-api.orig/examples/model-drivers/ex_model_driver_P_LJ/CMakeLists.txt
+++ kim-api/examples/model-drivers/ex_model_driver_P_LJ/CMakeLists.txt
@@ -39,7 +39,7 @@
 
 cmake_minimum_required(VERSION 3.4)
 
-list(APPEND CMAKE_MODULE_PATH $ENV{KIM_API_V2_CMAKE_MODULES_DIR})
+list(APPEND CMAKE_MODULE_PATH $ENV{KIM_API_V2_CMAKE_MODULES_DIR} "/usr/share/kim-api-v2/cmake")
 find_package(KIM-API-V2 2.0 REQUIRED)
 if(NOT TARGET kim-api)
   project("${KIM_API_PROJECT_NAME}" VERSION "${KIM_API_VERSION}"
Index: kim-api/examples/model-drivers/ex_model_driver_P_Morse/CMakeLists.txt
===================================================================
--- kim-api.orig/examples/model-drivers/ex_model_driver_P_Morse/CMakeLists.txt
+++ kim-api/examples/model-drivers/ex_model_driver_P_Morse/CMakeLists.txt
@@ -39,7 +39,7 @@
 
 cmake_minimum_required(VERSION 3.4)
 
-list(APPEND CMAKE_MODULE_PATH $ENV{KIM_API_V2_CMAKE_MODULES_DIR})
+list(APPEND CMAKE_MODULE_PATH $ENV{KIM_API_V2_CMAKE_MODULES_DIR} "/usr/share/kim-api-v2/cmake")
 find_package(KIM-API-V2 2.0 REQUIRED)
 if(NOT TARGET kim-api)
   project("${KIM_API_PROJECT_NAME}" VERSION "${KIM_API_VERSION}"
Index: kim-api/examples/models/LennardJones612_UniversalShifted__MO_959249795837_003/CMakeLists.txt
===================================================================
--- kim-api.orig/examples/models/LennardJones612_UniversalShifted__MO_959249795837_003/CMakeLists.txt
+++ kim-api/examples/models/LennardJones612_UniversalShifted__MO_959249795837_003/CMakeLists.txt
@@ -39,7 +39,7 @@
 
 cmake_minimum_required(VERSION 3.4)
 
-list(APPEND CMAKE_MODULE_PATH $ENV{KIM_API_V2_CMAKE_MODULES_DIR})
+list(APPEND CMAKE_MODULE_PATH $ENV{KIM_API_V2_CMAKE_MODULES_DIR} "/usr/share/kim-api-v2/cmake")
 find_package(KIM-API-V2 2.0 REQUIRED)
 if(NOT TARGET kim-api)
   project("${KIM_API_PROJECT_NAME}" VERSION "${KIM_API_VERSION}"
Index: kim-api/examples/models/LennardJones_Ar/CMakeLists.txt
===================================================================
--- kim-api.orig/examples/models/LennardJones_Ar/CMakeLists.txt
+++ kim-api/examples/models/LennardJones_Ar/CMakeLists.txt
@@ -39,7 +39,7 @@
 
 cmake_minimum_required(VERSION 3.4)
 
-list(APPEND CMAKE_MODULE_PATH $ENV{KIM_API_V2_CMAKE_MODULES_DIR})
+list(APPEND CMAKE_MODULE_PATH $ENV{KIM_API_V2_CMAKE_MODULES_DIR} "/usr/share/kim-api-v2/cmake")
 find_package(KIM-API-V2 2.0 REQUIRED)
 if(NOT TARGET kim-api)
   project("${KIM_API_PROJECT_NAME}" VERSION "${KIM_API_VERSION}"
Index: kim-api/examples/models/Sim_LAMMPS_ADP_StarikovLopanitsynaSmirnova_2018_SiAu__SM_985135773293_000/CMakeLists.txt
===================================================================
--- kim-api.orig/examples/models/Sim_LAMMPS_ADP_StarikovLopanitsynaSmirnova_2018_SiAu__SM_985135773293_000/CMakeLists.txt
+++ kim-api/examples/models/Sim_LAMMPS_ADP_StarikovLopanitsynaSmirnova_2018_SiAu__SM_985135773293_000/CMakeLists.txt
@@ -39,7 +39,7 @@
 
 cmake_minimum_required(VERSION 3.4)
 
-list(APPEND CMAKE_MODULE_PATH $ENV{KIM_API_V2_CMAKE_MODULES_DIR})
+list(APPEND CMAKE_MODULE_PATH $ENV{KIM_API_V2_CMAKE_MODULES_DIR} "/usr/share/kim-api-v2/cmake")
 find_package(KIM-API-V2 2.0 REQUIRED)
 if(NOT TARGET kim-api)
   project("${KIM_API_PROJECT_NAME}" VERSION "${KIM_API_VERSION}"
Index: kim-api/examples/models/ex_model_Ar_P_LJ/CMakeLists.txt
===================================================================
--- kim-api.orig/examples/models/ex_model_Ar_P_LJ/CMakeLists.txt
+++ kim-api/examples/models/ex_model_Ar_P_LJ/CMakeLists.txt
@@ -39,7 +39,7 @@
 
 cmake_minimum_required(VERSION 3.4)
 
-list(APPEND CMAKE_MODULE_PATH $ENV{KIM_API_V2_CMAKE_MODULES_DIR})
+list(APPEND CMAKE_MODULE_PATH $ENV{KIM_API_V2_CMAKE_MODULES_DIR} "/usr/share/kim-api-v2/cmake")
 find_package(KIM-API-V2 2.0 REQUIRED)
 if(NOT TARGET kim-api)
   project("${KIM_API_PROJECT_NAME}" VERSION "${KIM_API_VERSION}"
Index: kim-api/examples/models/ex_model_Ar_P_MLJ_Fortran/CMakeLists.txt
===================================================================
--- kim-api.orig/examples/models/ex_model_Ar_P_MLJ_Fortran/CMakeLists.txt
+++ kim-api/examples/models/ex_model_Ar_P_MLJ_Fortran/CMakeLists.txt
@@ -39,7 +39,7 @@
 
 cmake_minimum_required(VERSION 3.4)
 
-list(APPEND CMAKE_MODULE_PATH $ENV{KIM_API_V2_CMAKE_MODULES_DIR})
+list(APPEND CMAKE_MODULE_PATH $ENV{KIM_API_V2_CMAKE_MODULES_DIR} "/usr/share/kim-api-v2/cmake")
 find_package(KIM-API-V2 2.0 REQUIRED)
 if(NOT TARGET kim-api)
   project("${KIM_API_PROJECT_NAME}" VERSION "${KIM_API_VERSION}"
Index: kim-api/examples/models/ex_model_Ar_P_Morse/CMakeLists.txt
===================================================================
--- kim-api.orig/examples/models/ex_model_Ar_P_Morse/CMakeLists.txt
+++ kim-api/examples/models/ex_model_Ar_P_Morse/CMakeLists.txt
@@ -39,7 +39,7 @@
 
 cmake_minimum_required(VERSION 3.4)
 
-list(APPEND CMAKE_MODULE_PATH $ENV{KIM_API_V2_CMAKE_MODULES_DIR})
+list(APPEND CMAKE_MODULE_PATH $ENV{KIM_API_V2_CMAKE_MODULES_DIR} "/usr/share/kim-api-v2/cmake")
 find_package(KIM-API-V2 2.0 REQUIRED)
 if(NOT TARGET kim-api)
   project("${KIM_API_PROJECT_NAME}" VERSION "${KIM_API_VERSION}"
Index: kim-api/examples/models/ex_model_Ar_P_Morse_07C/CMakeLists.txt
===================================================================
--- kim-api.orig/examples/models/ex_model_Ar_P_Morse_07C/CMakeLists.txt
+++ kim-api/examples/models/ex_model_Ar_P_Morse_07C/CMakeLists.txt
@@ -39,7 +39,7 @@
 
 cmake_minimum_required(VERSION 3.4)
 
-list(APPEND CMAKE_MODULE_PATH $ENV{KIM_API_V2_CMAKE_MODULES_DIR})
+list(APPEND CMAKE_MODULE_PATH $ENV{KIM_API_V2_CMAKE_MODULES_DIR} "/usr/share/kim-api-v2/cmake")
 find_package(KIM-API-V2 2.0 REQUIRED)
 if(NOT TARGET kim-api)
   project("${KIM_API_PROJECT_NAME}" VERSION "${KIM_API_VERSION}"
Index: kim-api/examples/simulators/ex_test_Ar_fcc_cluster/CMakeLists.txt
===================================================================
--- kim-api.orig/examples/simulators/ex_test_Ar_fcc_cluster/CMakeLists.txt
+++ kim-api/examples/simulators/ex_test_Ar_fcc_cluster/CMakeLists.txt
@@ -39,7 +39,7 @@
 
 cmake_minimum_required(VERSION 3.4)
 
-list(APPEND CMAKE_MODULE_PATH $ENV{KIM_API_V2_CMAKE_MODULES_DIR})
+list(APPEND CMAKE_MODULE_PATH $ENV{KIM_API_V2_CMAKE_MODULES_DIR} "/usr/share/kim-api-v2/cmake")
 find_package(KIM-API-V2 2.0 REQUIRED)
 if(NOT TARGET kim-api)
   project("${KIM_API_PROJECT_NAME}" VERSION "${KIM_API_VERSION}"
Index: kim-api/examples/simulators/ex_test_Ar_fcc_cluster_cpp/CMakeLists.txt
===================================================================
--- kim-api.orig/examples/simulators/ex_test_Ar_fcc_cluster_cpp/CMakeLists.txt
+++ kim-api/examples/simulators/ex_test_Ar_fcc_cluster_cpp/CMakeLists.txt
@@ -39,7 +39,7 @@
 
 cmake_minimum_required(VERSION 3.4)
 
-list(APPEND CMAKE_MODULE_PATH $ENV{KIM_API_V2_CMAKE_MODULES_DIR})
+list(APPEND CMAKE_MODULE_PATH $ENV{KIM_API_V2_CMAKE_MODULES_DIR} "/usr/share/kim-api-v2/cmake")
 find_package(KIM-API-V2 2.0 REQUIRED)
 if(NOT TARGET kim-api)
   project("${KIM_API_PROJECT_NAME}" VERSION "${KIM_API_VERSION}"
Index: kim-api/examples/simulators/ex_test_Ar_fcc_cluster_fortran/CMakeLists.txt
===================================================================
--- kim-api.orig/examples/simulators/ex_test_Ar_fcc_cluster_fortran/CMakeLists.txt
+++ kim-api/examples/simulators/ex_test_Ar_fcc_cluster_fortran/CMakeLists.txt
@@ -39,7 +39,7 @@
 
 cmake_minimum_required(VERSION 3.4)
 
-list(APPEND CMAKE_MODULE_PATH $ENV{KIM_API_V2_CMAKE_MODULES_DIR})
+list(APPEND CMAKE_MODULE_PATH $ENV{KIM_API_V2_CMAKE_MODULES_DIR} "/usr/share/kim-api-v2/cmake")
 find_package(KIM-API-V2 2.0 REQUIRED)
 if(NOT TARGET kim-api)
   project("${KIM_API_PROJECT_NAME}" VERSION "${KIM_API_VERSION}"
